src/components/MovieForm/MovieForm.tsx

```diff
@@ -101,18 +107,23 @@ function MovieForm({movie, genres, onSubmit}: MovieFormProps) {
                                 dateFormat='yyyy-MM-dd'/>
                 )}
             />
+            <ErrorMessage errors={errors} name="releaseDate" render={({ message }) => <p className="error-message">{message}</p>}/>
           </div>
         </div>
 
         <div className="row mb-3">
           <div className="col-7">
             <label htmlFor="posterUrl">Movie Url</label>
```

Comment:
Won't block the PR because of this, but all the individual inputs together with labels and errors seem like a good candidates for their own components, so `MovieForm` is just a layout container.
https://react-hook-form.com/advanced-usage

Reply:
I will try to extract it into individual components

Reply:
Extracted all form elements into individual components

Reply:
Nice job, looks much better this way!